icuSources/common/ubidiimp.h

```diff
@@ -239,17 +239,17 @@ enum {
 
 /* InsertPoints structure for noting where to put BiDi marks ---------------- */
 
-typedef struct _Point {
```

Comment:
Is this no longer an issue now?

Reply:
Confirmed in private that this should no longer be an issue since #34

Reply:
Nope